Jenkinsfile-e2e

```diff
@@ -1,152 +1,69 @@
-                  colour: 'good',
-                  emoji: '🤟'
-                ]
-                notifySlack(messageParameters, channels.simorgh)
-                break
-              case 'UNSTABLE':
-                def messageParameters = [
-                  buildStatus: 'UNSTABLE',
-                  env: 'Local and Production',
-                  branch: env.BRANCH_NAME,
-                  colour: 'danger',
-                  emoji: '😱',
-                ]
-                notifySlack(messageParameters, channels.liveAlarms)
-                break
-              case 'ABORTED':
-                def messageParameters = [
-                  buildStatus: 'ABORTED',
-                  env: 'Local and Production',
-                  branch: env.BRANCH_NAME,
-                  colour: 'warning',
-                  emoji: '🙀',
-                ]
-                notifySlack(messageParameters, channels.liveAlarms)
-                break
-              default:
-                if (stageHasFailed == false) {
-                  def messageParameters = [
-                    buildStatus: 'Unknown build failure',
-                    env: 'Local and Production',
-                    branch: env.BRANCH_NAME,
-                    colour: 'danger',
-                    emoji: '😱',
-                  ]
-                  notifySlack(messageParameters, channels.liveAlarms)
+            docker.image("${nodeImage}").inside('--ipc host') {
+                try {
+                    Simorgh.installNodeModules()
+                    parallel(
+                        'FrontPage E2Es': {
+                            Simorgh.runE2Es('live', false, 'cypress/integration/pages/frontPage/index.js')
+                        },
+                        'Article E2Es': {
+                            sh script: 'sleep 5', label: 'Sleeping, allowing the first cypress instance to launch and control Xvfb'
+                            Simorgh.runE2Es('live', false, 'cypress/integration/pages/articles/index.js')
+                        },
+                        'ErrorPage E2Es': {
+                            sh script: 'sleep 5', label: 'Sleeping, allowing the first cypress instance to launch and control Xvfb'
+                            Simorgh.runE2Es('live', false, 'cypress/integration/pages/errorPage404/index.js')
+                        },
+                        'Live Radio E2Es': {
+                            sh script: 'sleep 5', label: 'Sleeping, allowing the first cypress instance to launch and control Xvfb'
+                            Simorgh.runE2Es('live', false, 'cypress/integration/pages/liveRadio/index.js')
+                        },
+                        'MAP E2Es': {
+                            sh script: 'sleep 5', label: 'Sleeping, allowing the first cypress instance to launch and control Xvfb'
+                            Simorgh.runE2Es('live', false, 'cypress/integration/pages/mediaAssetPage/index.js')
+                        },
```

Comment:
I like the idea of parallelising these but by naming each file explicitly here isn't there a risk is that later someone adds a new set of tests or renames an existing one and we don't notice for ages that it isn't being run on Jenkins, because they were assuming we just run "all the tests"?

Reply:
...and presumably the list will need duplicating in `e2e-test` too?

Reply:
There is that risk yes, I was speaking with Benjamin about this earlier.

I think a good starting solution to this would be to update our docs about how and where we run e2e's and to add core as code owners to the spec dir, this way it will be on the core pod to ensure any new spec is added to the pipeline.

Reply:
The list is already duplicated in the e2e-test Jenkinsfile.

A future improvement to this would be to see if we can dynamically build this list of stages from files in the spec dir, so by simply adding a new spec you would essentially be telling the pipeline to run that spec.